firebase-database/src/androidMain/kotlin/dev/gitlive/firebase/database/ServerValue.kt

```diff
@@ -0,0 +1,21 @@
+package dev.gitlive.firebase.database
+
+import kotlinx.serialization.Serializable
+
+private typealias NativeServerValue = com.google.firebase.database.ServerValue
+
+/** Represents a Firebase ServerValue. */
+@Serializable(with = ServerValueSerializer::class)
+actual class ServerValue internal actual constructor(
```

Comment:
using a wrapper class allows specifying a custom serializer and also ensures correct equality checks in JS